### PR TITLE
LapCap data retrieve GET API call

### PR DESCRIPTION
**Pull request by rekhadhankhar2018**

### CODE REVIEW

**Review by johnbenjamin-defra — APPROVED** [review]
Approved with minor comment

File: src/EPR.Calculator.API.UnitTests/CreateDefaultParameterSettingMapperTest.cs [hunk 1]
```diff
@@ -11,55 +14,50 @@ namespace api.Tests.Controllers
     public class CreateDefaultParameterSettingMapperTest : BaseControllerTest
     {
         [TestMethod]
-        public void Check_TheResult_IsNotNullOf_ResultSet_WithDefaultSchemeParametersDto_WithCorrectYear()
+        public void Check_TheResult_Parmeter_Are_Equal_IsNotNullOf_ResultSet_WithDefaultSchemeParametersDto_WithCorrectYearr()
```
[the remark this answers is not in the document]
Comment (rekhadhankhar2018) [reply src/EPR.Calculator.API.UnitTests/CreateDefaultParameterSettingMapperTest.cs]: thanks John i have fixed that now

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by udaydefra — CHANGES_REQUESTED** [review]
The mapping logic seems to be incorrect as we discussed in the call. Could you please fix it. Please change the unit tests also. @rekhadhankhar2018

**Review by udaydefra — APPROVED** [review]
LGTM

**Review by johnbenjamin-defra — APPROVED** [review]

**Pull request merged**